extension/js/common/api/email-provider/gmail/google.ts

```diff
@@ -10,6 +10,32 @@ import { GmailRes } from './gmail-parser.js';
 import { GoogleAuth } from './google-auth.js';
 import { Serializable } from '../../../platform/store/abstract-store.js';
 
+type GoogleUserProfile = {
+  resourceName: string;
+  etag: string;
+  names: GoogleName[];
+};
```

Comment:
We probably don't need this full object, let's use only needed fields and move it to [GmailRes](https://github.com/FlowCrypt/flowcrypt-browser/blob/master/extension/js/common/api/email-provider/gmail/gmail-parser.ts#L15):

```js
export type GoogleProfileNames = {
    names: {
      metadata: {
        primary: boolean;
      };
      displayName: string;
    };
  };
```